api/send_test.go

```diff
@@ -136,12 +135,14 @@ func TestSendMessage(t *testing.T) {
 
 	t.Run("invalid embed", func(t *testing.T) {
 		var data = SendMessageData{
-			Embed: &discord.Embed{
-				// max 256
-				Title: spaces(257),
-			},
+			Embeds: []discord.Embed{},
```

Comment:
It seems like the test isn't fixed properly here, causing the CI to fail.

Reply:
Fixed.